core/event/event-manager.js

```diff
@@ -1642,7 +1638,7 @@ if (typeof window !== "undefined") { // client-side
                     return (this.memory[identifier] && (this.memory[identifier].size > 0));
                 },
                 storeEvent: function (mutableEvent) {
-                    var isBrowserSupportPointerEvents = this._isBrowserSupportPointerEvents,
+                    var isBrowserSupportPointerEvents = currentEnvironment.isBrowserSupportPointerEvents,
```

Comment:
currentEnvironment should be a property on eventManager rather than a closure, storeEvent is used with high frequency so preferable to avoid a closure here